cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -479,3 +480,58 @@ def _is_migration_started():
     for vm in vms:
         soft_assert(request_details_list.is_successful(vm) and
          not request_details_list.is_errored(vm))
+
+
+@pytest.mark.parametrize('form_data_vm_obj_single_datastore', [['nfs', 'nfs', rhel7_minimal]],
+                         indirect=True)
+def test_migration_policy_tag(request, appliance, v2v_providers, host_creds, conversion_tags,
+                              form_data_vm_obj_single_datastore, soft_assert):
+    """Test policy to prevent source VM from starting if migration is complete"""
+    infrastructure_mapping_collection = appliance.collections.v2v_mappings
+    # form_data_vm_obj_single_datastore has form_data at [0]
+    mapping = infrastructure_mapping_collection.create(form_data_vm_obj_single_datastore[0])
+
+    @request.addfinalizer
+    def _cleanup():
+        infrastructure_mapping_collection.delete(mapping)
+
+    migration_plan_collection = appliance.collections.v2v_plans
+    # form_data_vm_obj_single_datastore has vm_obj list at [1]
+    vm_obj = form_data_vm_obj_single_datastore[1][0]
+    migration_plan = migration_plan_collection.create(
+        name="plan_{}".format(fauxfactory.gen_alphanumeric()),
+        description="desc_{}".format(fauxfactory.gen_alphanumeric()),
+        infra_map=mapping.name,
+        vm_list=[vm_obj],
+        start_migration=True)
+
+    # explicit wait for spinner of in-progress status card
+    view = appliance.browser.create_view(navigator.get_class(migration_plan_collection, 'All').VIEW)
+    wait_for(func=view.progress_card.is_plan_started, func_args=[migration_plan.name],
+        message="migration plan is starting, be patient please", delay=5, num_sec=150,
+        handle_exception=True)
+
+    # wait until plan is in progress
+    wait_for(func=view.plan_in_progress, func_args=[migration_plan.name],
+        message="migration plan is in progress, be patient please",
+        delay=5, num_sec=3600)
```

Comment:
Required:
we are trying to check for `num_sec=3600` please provide suitable `delay`. I mean `5` is too less. what do you think?

Reply:
@digitronik agree, 15s is better I guess, comment back in case.